ci_framework/plugins/action/discover_latest_image.py

```diff
@@ -102,12 +92,15 @@ def run(self, tmp=None, task_vars=None):
 
         extracted_sha = sha256sum['content'].split('=')[1].strip()
 
-        result['data'] = {
-            'image_name': latest_image,
-            'image_url': f'{base_url}/{latest_image}',
-            'sha256': extracted_sha,
+        result = {
+            'success': True,
+            'changed': True,
+            'error': '',
```

Comment:
Is this really needed?  It wasn't here in the previous version. I cannot find any doc about the interface of an action module in regard of the return value. Haven't seen any action module that returns this key.